about.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/main.css" />
+    <title>Document</title>
+  </head>
+  <body>
+      <div>
+      <main class="about">
+          <div id="top-social-media">
+            <span class="icon"><img src="./Images/Icons/facebook_f.svg" alt=""></span>
+            <span class="icon"><img src="./Images/Icons/twitter_squared.svg" alt=""></span>
+            <p>English</p>
+            <p>My Page</p>
+            <p>Logout</p>
+          </div>
+        <header>
+          <div id="drop-menu">
+            <img src="Images/Icons/humberge.svg" alt="show menu" />
+          </div>
+          <div id="logo">
+            <a href="./"><img src="./Images/Icons/logo.svg" alt="" /></a>
+          </div>
+          <nav class="hide">
```

Comment:
Well done! Both pages contain the same navbar and footer ✔

Reply:
All elements of your navbar are positioned as expected 🔥

Reply:
@mateo951 Thank you.